In [1]:
#Importing Library
import pandas as pd

In [2]:
#importing dataset
dataset=pd.read_csv("PrePlacement.csv")

In [3]:
#viewing dataset
dataset

Unnamed: 0  sl_no gender  ssc_p    ssc_b  hsc_p    hsc_b     hsc_s  \
0             0      1      M  67.00   Others  91.00   Others  Commerce   
1             1      2      M  79.33  Central  78.33   Others   Science   
2             2      3      M  65.00  Central  68.00  Central      Arts   
3             3      4      M  56.00  Central  52.00  Central   Science   
4             4      5      M  85.80  Central  73.60  Central  Commerce   
..          ...    ...    ...    ...      ...    ...      ...       ...   
210         210    211      M  80.60   Others  82.00   Others  Commerce   
211         211    212      M  58.00   Others  60.00   Others   Science   
212         212    213      M  67.00   Others  67.00   Others  Commerce   
213         213    214      F  74.00   Others  66.00   Others  Commerce   
214         214    215      M  62.00  Central  58.00   Others   Science   

     degree_p   degree_t workex  etest_p specialisation  mba_p      status  \
0       58.00   Sci&Tech     No     55.0         Mkt&HR  58.80      Placed   
1       77.48   Sci&Tech    Yes     86.5        Mkt&Fin  66.28      Placed   
2       64.00  Comm&Mgmt     No     75.0        Mkt&Fin  57.80      Placed   
3       52.00   Sci&Tech     No     66.0         Mkt&HR  59.43  Not Placed   
4       73.30  Comm&Mgmt     No     96.8        Mkt&Fin  55.50      Placed   
..        ...        ...    ...      ...            ...    ...         ...   
210     77.60  Comm&Mgmt     No     91.0        Mkt&Fin  74.49      Placed   
211     72.00   Sci&Tech     No     74.0        Mkt&Fin  53.62      Placed   
212     73.00  Comm&Mgmt    Yes     59.0        Mkt&Fin  69.72      Placed   
213     58.00  Comm&Mgmt     No     70.0         Mkt&HR  60.23      Placed   
214     53.00  Comm&Mgmt     No     89.0         Mkt&HR  60.22  Not Placed   

       salary  
0    270000.0  
1    200000.0  
2    250000.0  
3         0.0  
4    425000.0  
..        ...  
210  400000.0  
211  275000.0  
212  295000.0  
213  204000.0  
214       0.0  

[215 rows x 16 columns]

In [4]:
#to test the Analysis of Variance between etest_p and mba_p
import scipy.stats as stats
stats.f_oneway(dataset["etest_p"],dataset["mba_p"])

F_onewayResult(statistic=98.64487057324706, pvalue=4.672547689133573e-21)

In [5]:
#Since the p-value is greater than the significance level of 0.05, we fail to reject the null hypothesis. 
#reject null hypothesis P<0.05
#accept null hypothesis P>0.05

Assignment Question : Test the Analysis of Variance between etest_p and mba_p at signifancelevel 5%.(Make decision using Hypothesis Testing)

Resolution :  By using above code, we found pvalue=4.67. Since the p-value is greater than the significance level of 0.05, we fail to reject the null hypothesis. 